test/test-initialization-errors.js

```diff
@@ -142,7 +142,7 @@ it('errors are thrown in connectedCallback when template result fails to render
   try {
     el.connectedCallback();
   } catch (error) {
-    const expected = ' — Invalid template for "TestElement" at path "test-element-4[id="testing"][class="foo bar"][boolean][variation="primary"]"';
+    const expected = 'Invalid template for "TestElement" / <test-element-4> at path "test-element-4[id="testing"][class="foo bar"][boolean][variation="primary"]".';
```

Comment:
Seeing something like `<test-element-4>` is often more useful than the actual class name for finding the file-in-question. I honestly don’t know why we hadn’t done this yet 🤔